Solutions/AristaAwakeSecurity/Workbooks/WorkbooksMetadata.json

```diff
@@ -0,0 +1,15 @@
+[
+    {
```

Comment:
As the workbook is to be deployed as part of Solution, we don't need to add WorkbookMetadata.json file... Please delete this file.... Thanks!!!

Reply:
File was removed.